pages/Butcher-Androids.md

```diff
@@ -0,0 +1,18 @@
+An Android that specializes in killing hostile mobs or/and animals.
+
+## Obtaining
+Butcher Androids can be crafted in an [Enhanced Crafting Table](https://github.com/Slimefun/Slimefun4/wiki/Enhanced-Crafting-Table).
+
+## Usage
+Butcher Androids can be programmed to kill hostile mobs or/and animals.  
+This can also can be paired with an [EXP Collector](https://github.com/Slimefun/Slimefun4/wiki/EXP-Collector) for automated EXP Farm.  
```

Comment:
```suggestion
This can also can be paired with an [EXP Collector](https://github.com/Slimefun/Slimefun4/wiki/EXP-Collector) for an automated EXP Farm.  
```